en/includes/references/grant-types.md

```diff
@@ -1,49 +1,111 @@
+Grant types in OAuth 2.0 are defined as the methods used by a client to obtain an access token from the identity provider. {{product_name}} supports the following grant types. Each grant type is designed for a specific use case and supports different parameters.
 
-The grant types supported by {{ product_name }} applications are as follows:
-
-**OAuth 2.0 grants**
+#### OAuth 2.0 grants
 
 - [Authorization code grant](#authorization-code-grant)
 - [Refresh token grant](#refresh-token-grant)
 - [Client credentials grant](#client-credentials-grant)
 - [Implicit grant](#implicit-grant)
 - [Password grant](#password-grant)
+{% if product_name == "WSO2 Identity Server" %}
+- [Device authorization grant](#device-authorization-grant)
+{% endif %}
 - [Token exchange grant](#token-exchange-grant)
 - [SAML 2.0 bearer grant](#saml-20-bearer-grant)
 
-**{{ product_name }}'s custom grants**
+#### {{product_name}}'s custom grants
 
 - [Organization switch grant](#organization-switch-grant)
 
+The following sections explore each grant type in further detail.
+
 {{grant_type_configs_note}}
 
 ## Authorization code grant
 
-The Authorization code flow provides a secure way for a client application to obtain an access token without exposing the user's credentials to the client application. The user only authenticates with the authorization server, which then issues an authorization code that can be exchanged for an access token.
-
-This helps to protect the user's credentials and prevents them from being compromised by malicious client applications.
+The Authorization code flow is a secure method for clients to obtain an access token by exchanging an authorization code received through the front channel. As the access token is directly transmitted to the client via a secure back channel, it is not exposed to the user's user agent (e.g. web browser) reducing its risk of being intercepted by a malicious party.
 
 The following diagram shows how the authorization code flow works.
 
 ![How the authorization code grant works]({{base_path}}/assets/img/references/grants/authorization-code.png)
 
 1. The user visits the client application and requests for login through {{ product_name }}.
-2. The client application redirects the authorization code request to {{ product_name }}.
+2. The client application redirects the authorization code request to {{ product_name }} (front channel).
+
+
+    === "Request format (/authorize)"
+
+        ``` bash
+        {{base_url}}/oauth2/authorize
+        ?response_type=code
+        &client_id=<CLIENT_ID>
+        &redirect_uri=<REDIRECT_URI>
+        ```
+
+    === "Sample request (/authorize)"
+
+        ```bash
+        {{base_url_example}}/oauth2/authorize
+        ?response_type=code
+        &client_id=cHuGSLCm77ApRFcsdyh_4sdFU2XYa
+        &redirect_uri=https://localhost:3000
+        ```
+
 3. {{ product_name }} prompts the user to enter credentials.
 4. The user enters the credentials.
```

Comment:
This is not just the user credentials login, what ever the configured login options of the application will be prompted.
user needs to authenticate from one login option

Better to reword step 3 and 4